site/content/zh/contributions/DOCS.md

```diff
@@ -5,8 +5,6 @@ description: "本节讲述 Envoy Gateway 文档的开发工作。"
 
 {{% alert title="注意" color="warning" %}}
 我们已将 Envoy Gateway 文档从 **Sphinx** 迁移到 **Hugo**。
-
-阅读博客：[Welcome to new website!](/blog/2023/10/08/welcome-to-new-website/)
```

Comment:
remove here just to pass doc-lint in ci, shall undo it when translating chinese doc

Reply:
I will remember it and add it back with the sync to zh PR.